### PR TITLE
FINALLY.

### PR DESCRIPTION
**Pull request by PoorPocketsMcNewHold**
After two hours of translation, Hopefully it has been worth the wait.

### CODE REVIEW

File: lang/french.json [hunk 1]
```diff
@@ -236,19 +256,174 @@
...
+    "sydneyhud_hudlist_not_ignore_buff_painkillers_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_quick_fix_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_uppers_cooldown" : "Temps de recharge des remontants",
+    "sydneyhud_hudlist_not_ignore_buff_uppers_cooldown_desc" : "Les debuffs composite, s'afficheront en tant que qu'une partie de l'icone principale de buff même désactivé",
+    "sydneyhud_hudlist_not_ignore_buff_bullseye_cooldown" : "Temps de recharge de Dans l'mille",
+    "sydneyhud_hudlist_not_ignore_buff_die_hard_desc" : "Aussi inclus dans le buff composite de réduction de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_overkill_desc" : "Aussi inclus dans le buff composite de boost de dégats",
+    "sydneyhud_hudlist_not_ignore_buff_shock_and_awe_desc" : "Buff d'équipe",
+    "sydneyhud_hudlist_not_ignore_buff_underdog_desc" : "Aussi inclus dans le buff composite de réduction de dégats et de boost de dégats des armes",
+    "sydneyhud_hudlist_not_ignore_buff_unseen_strike_cooldown" : "Unseen Strike Cooldown",
```
Comment (dominikvyhnak) [inline lang/french.json]: Somehow you missed this one.
Unseen Strike in French is "Frappe Invisible"

File: lang/french.json [hunk 2]
```diff
@@ -236,19 +256,174 @@
...
+    "sydneyhud_hudlist_show_buffs_desc" : "Décocher pour désactiver l'affichage de tout les buffs (Si activé, les buffs individuels peuvent être activés/désactivés ci-dessous)",
+    "sydneyhud_hudlist_show_player_actions" : "Montrer les actions du joueur",
+    "sydneyhud_hudlist_show_player_actions_desc" : "Décocher pour désactiver l'affichage de toutes les actions effectuées (Si activé, les actions individuelles peuvent être activés/désactivés ci-dessous)",
+    "sydneyhud_hudlist_not_ignore_item_crowbar" : "Pieds de biche",
+    "sydneyhud_hudlist_not_ignore_item_courier" : "Paquet de coursier de Gages",
+    "sydneyhud_hudlist_not_ignore_item_keycard" : "Carte-clé",
+    "sydneyhud_hudlist_not_ignore_item_meth_ingredients" : "Ingrédients de Meth",
+    "sydneyhud_hudlist_not_ignore_item_planks" : "Planches",
+    "sydneyhud_hudlist_not_ignore_item_secret_item" : "Items secrets",
+    "sydneyhud_hudlist_not_ignore_item_secret_item_desc" : "Différents items spéciaux tel que les bagues dans Birth of Sky et les bouteilles sur le Biker Heist",
```
Comment (dominikvyhnak) [inline lang/french.json]: Birth of Sky in French is "Genèse Celéste"
Biker Heist in French is "À Pleinz Gaz"

File: lang/french.json [hunk 3]
```diff
@@ -294,6 +469,7 @@
     "sydneyhud_save_preplanning_desc" : "Sauvegarde votre planification",
     "sydneyhud_show_assault_states" : "Montrer les États d'assaut",
     "sydneyhud_show_assault_states_desc" : "Montre les États d'assaut (Contrôle, Anticipation, Développement, Support et Retrait) dans la boîte d'assaut\nCette option n'est pas disponible en cas d'assaut sans fin et quand le capitaine est présent",
+      "sydneyhud_show_assault_states_disabled_desc" : "Le menu est désactivé et les options dans le menu ne peuvent être changés pendant que BAI est activé\nChangez ces options dans le menu de BAI",
```
Comment (dominikvyhnak) [inline lang/french.json]: This string was replaced with "sydneyhud_hud_tweaks_assault_disabled_desc" and also the string description was changed.

File: lang/french.json [hunk 4]
```diff
@@ -236,19 +256,174 @@
...
+    "sydneyhud_hudlist_not_ignore_buff_unseen_strike_cooldown_desc" : "Debuff composite, s'affichera en tant que qu'une partie de l'icone principale de buff même désactivé",
+    "sydneyhud_hudlist_not_ignore_buff_berserker_desc" : "Aussi inclus dans le buff composite de boost de dégats d'armes et de mélée",
     "sydneyhud_hudlist_not_ignore_buff_bloodthirst_basic" : "Soif De Sang (Basique)",
     "sydneyhud_hudlist_not_ignore_buff_bloodthirst_aced" : "Soif De Sang (As)",
     "sydneyhud_hudlist_not_ignore_buff_running_from_death_basic" : "Course Contre La Mort (Basique)",
+    "sydneyhud_hudlist_not_ignore_buff_bloodthirst_basic_desc" : "Aussi inclus dans le buff composite de dégats de mélée",
     "sydneyhud_hudlist_not_ignore_buff_running_from_death_aced" : "Course Contre La Mort (As)",
-    
-    "sydneyhud_hud_tweaks": "Amélioration de l'HUD",
+    "sydneyhud_hudlist_not_ignore_buff_up_you_go_desc" : "Also included in damage reduction composite buff",
```
Comment (dominikvyhnak) [inline lang/french.json]: This string is also not translated.

File: lang/french.json [hunk 5]
```diff
@@ -90,14 +103,16 @@
...
+    "hud_phase" : "Phase d'assaut: ",
     "hud_control" : "Contrôle",
     "hud_anticipation" : "Anticipation",
     "hud_build" : "Développement",
     "hud_sustain" : "Support",
     "hud_fade" : "Retrait",
     "hud_overdue" : "Retard",
     "hud_time_left" : "Temps restant: ",
-    "hud_spawns_left" : "Spawns restants: ",
+    "hud_spawns_left" : "Apparitions restantes: ",
```
[the remark this answers is not in the document]
Comment (PoorPocketsMcNewHold) [reply lang/french.json]: Spawns is commonly used in french as well, but I've decided to fight more anglicism in my french translations, It's the translation of the word "appearance", in a way to say "How many remaining times enemies will appear ?". So, I let you the choice to replace it or not, as I've said, "spawn" is still commonly used in french gaming vocabulary.

File: lang/french.json (hunk 3 above)
Comment (PoorPocketsMcNewHold) [reply lang/french.json]: So, I suppose I need to remove it ?

File: lang/french.json (hunk 2 above)
Comment (PoorPocketsMcNewHold) [reply lang/french.json]: Forgot that the map names were localized. It's been a while, and I've immediately only remembered how I was habitually using the original names, and so, forgetting to check for the original localized ones.

File: lang/french.json (hunk 1 above)
Comment (PoorPocketsMcNewHold) [reply lang/french.json]: Yep, Indeed. Fixed

File: lang/french.json (hunk 4 above)
Comment (PoorPocketsMcNewHold) [reply lang/french.json]: I've been a bit bored with copy pasting this common sentence that I've may I've skip one like you can see 😒

File: lang/french.json [hunk 6]
```diff
@@ -203,20 +221,22 @@
...
-    "skip_loot_screen_desc": "Passe les écrans de loot",
+    "skip_loot_screen": "Passe l'écran de butin",
+    "skip_loot_screen_desc": "Passe les écrans de butin",
     "skip_stat_screen": "Passe l'écran des statistiques",
     "skip_stat_screen_desc": "Passe les écrans des statistiques",
     "stat_screen_skip": "Délai pour passer l'écran des stats",
     "stat_screen_skip_desc": "Passe l'écran des statistiques après X secondes",
-    "swansong_effect": "Effet \"Chant du signe\"",
-    "swansong_effect_desc": "Quand vous activez la compétence \"Chant du signe\", un effet clignotant visuel apparaît",
+    "swansong_effect": "Effet Chant du signe,
```
[the remark this answers is not in the document]
Comment (PoorPocketsMcNewHold) [reply lang/french.json]: Been a bit confused with the old syntax that I've had to remove, that I've accidentally remove another ".

File: lang/french.json (hunk 3 above)
Comment (dominikvyhnak) [reply lang/french.json]: More like replace that string with this:

**Pull request merged**